.github/workflows/notify-release.yml

```diff
@@ -18,5 +18,5 @@ jobs:
               {
                 "avatar_url":"https://avatars.githubusercontent.com/u/98603954?v=4",
                 "username":"Bot Anik",
-                "content":"🚨 A new version of @${{github.repository}} ${{ github.event.tagName }} has been released! 🎉 \n \n 👉 Changelog: https://github.com/${{ github.repository }/releases/tag/${{ github.event.tagName }} \n 👉 Documentation: https://ui.okp4.space/?path=/story/welcome--page \n 👉 Official repo: https://github.com/${{ github.repository } \n 👉 NPM link: https://www.npmjs.com/package/@okp4/ui"
+                "content":"🚨 A new version of @${{github.repository}} ${{ github.event.tagName }} has been released! 🎉 \n \n 👉 Changelog: https://github.com/${{ github.repository }}/releases/tag/${{ github.event.tagName }} \n 👉 Documentation: https://ui.okp4.space/?path=/story/welcome--page \n 👉 Official repo: https://github.com/${{ github.repository }} \n 👉 NPM link: https://www.npmjs.com/package/@okp4/ui"
```

Comment:
Please, parametrize the NPM link as well (t should have been done in the PR #512):

```suggestion
                "content":"🚨 A new version of @${{github.repository}} ${{ github.event.tagName }} has been released! 🎉 \n \n 👉 Changelog: https://github.com/${{ github.repository }}/releases/tag/${{ github.event.tagName }} \n 👉 Documentation: https://ui.okp4.space/?path=/story/welcome--page \n 👉 Official repo: https://github.com/${{ github.repository }} \n 👉 NPM link: https://www.npmjs.com/package/@${{ github.repository }}"
```

thx.

Reply:
Please, if you don't mind remove unnecessary spaces between `\n` also.